packages/teams-js/test/private/files.spec.ts

```diff
@@ -18,59 +17,43 @@ describe('files', () => {
     utils.mockWindow.parent = utils.parentWindow;
 
     // Set a mock window for testing
-    app._initialize(utils.mockWindow);
```

Comment:
Fine to leave this as is but in case anyone else is looking and curious: app._initialize is called by _initialize so this change is actually the same